content/backmatter.tex

```diff
@@ -637,20 +651,22 @@ \section{Version 1.5}
 \begin{itemize}
 %
 \item Removed \FUNC{SHMEM\_CACHE}.
+\ChangelogRef{dep:shmem_cache}%
 %
 \item Deprecated \CTYPE{short} and \CTYPE{unsigned short} variants for
 \FUNC{shmem\_wait\_until} and \FUNC{shmem\_test}.
+\ChangelogRef{dep:short_ushort_typed_shmem_wait_until_and_test}%
 %
 \item Added \FUNC{shmem\_malloc\_with\_hints} interface and corresponding hints
 \CONST{SHMEM\_MALLOC\_ATOMICS\_REMOTE} and \CONST{SHMEM\_MALLOC\_SIGNAL\_REMOTE}.
-\\ See Section \ref{subsec:shmmallochint} and \ref{subsec:library_constants}.
+\ChangelogRef{subsec:shmmallochint, subsec:library_constants}%
 %
 \item Specified that team-based broadcast operations update the \VAR{dest} object on
 all \acp{PE}, including the root \ac{PE}.
-\\ See Section \ref{subsec:shmem_broadcast}.
+\ChangelogRef{subsec:shmem_broadcast}%
 %
 \item Deprecated active-set-based library constants and collective functions.
-\\ See Sections \ref{subsec:library_constants} and \ref{subsec:coll}.
+\ChangelogRef{dep:active_set_libconst_and_collectives, dep:shmem_barrier}%
```

Comment:
This removes the references to the library constants and collectives sections and replaces them with references to the deprecation annex. I'm wondering if we want to have both references?

Reply:
My motivation was that&mdash;historically&mdash;many deprecated items didn't have references at all and now that I'm adding them, having only a ref to the deprecation rationale made sense to me since the deprecated interfaces are listed in the rationales too.

Nonetheless, I'm okay with both references if we want to do it that way. I'd have to go back and review what relevant sections should be included for each deprecated item.

Reply:
This is a case where the APIs are still in the document, but marked as deprecated. In these cases, I think we should reference both the deprecation annex and the API definitions. Alternatively, we could reference the API definitions from the deprecation annex, but I don't think we are doing this right now.

Reply:
@BryantLam Any change needed here?

Reply:
All good. This was resolved in https://github.com/jdinan/specification/pull/15/commits/f025b115809c08582c1be869cbbaa0b48271194d.